src/objects/Error_Settings__c.object

```diff
@@ -23,6 +23,16 @@
         <trackTrending>false</trackTrending>
         <type>Checkbox</type>
     </fields>
+    <fields>
+        <fullName>DisableRecordDataHealthChecks__c</fullName>
```

Comment:
Since this new field is added, need to change [`UTIL_CustomSettingsFacade.getErrorSettingsForTests()`](https://github.com/SalesforceFoundation/NPSP/blob/master/src/classes/UTIL_CustomSettingsFacade.cls#L844) to set the new field.